lib/bolt/boltdir.rb

```diff
@@ -37,6 +38,17 @@ def initialize(path)
       @hiera_config = @path + 'hiera.yaml'
       @puppetfile = @path + 'Puppetfile'
       @rerunfile = @path + '.rerun.json'
+      @type = compute_type
+    end
+
+    def compute_type
```

Comment:
This misses the case where `--boltdir` was passed. 

Instead of recomputing this I think `find_boltdir` or whatever creates the instance should pass the "type" as an argument.

Reply:
Updated to pass the type to the initialize method.